scripts/gdb.py

```diff
@@ -174,6 +178,28 @@ def set_state(self, registers):
     for r, v in registers.items():
       gdb.execute("set ${} = {}".format(r,v))
 
+class Sgx_Add_Symbol_File (gdb.Command):
+  """
+  Given TCS address and ELF path, load enclave's symbols.
```

Comment:
“Given any address in the enclave and ELF path...”

Reply:
Thanks. Fixed in current push.

Reply:
Still says TCS in a few more places

Reply:
Also I think you can say "Specifying auto is equivalent to specifying `$rbx`"

Reply:
Changed the description in current push. Also removed TCS from the command syntax.